woo-components/src/components/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,48 @@
+const React = require('react');
+const { PropTypes } = React;
+
+class Pagination extends React.Component {
+
+  render () {
+    const PageLinks = ({amount}) => {
+      return (
+        <div>
+          {
+            Array(amount).fill().map((_, i) => {
+              return (
+                <button className='btn btn-default' key={i}
+                  onClick={() => this.props.onPagination(i * this.props.step, this.props.step)}>
+                  {i + 1}
+                </button>
+              );
+            })
+          }
+        </div>);
+    };
+
+    const pageCount = (
+      Math.floor(this.props.total / this.props.step) +
+      Math.min(this.props.total % this.props.step, 1)
+    );
+
+    return (
+      <div className='woo-pagination'>
+        <PageLinks amount={pageCount} />
+      </div>
+    );
+  }
+}
+
+Pagination.defaultProps = {
+  first: 0,
+  step: 10
+};
+
+Pagination.propTypes = {
```

Comment:
Do you think it would be possible to have different modes of pagination? Because not every app paginates the same way, especially for large data stores.
The different paginations I know about are:

- offset, limit
- page, perPage
- after, limit